plugins/chat/config/locales/server.en.yml

```diff
@@ -113,14 +113,16 @@ en:
           title: "Delete Message"
           description: "Delete the message so that users cannot see it."
         delete_and_agree:
-          title: "Delete Message"
+          title: "Ignore Flag and Delete Message"
+          description: "Ignore the flag by removing it from the queue and delete the message."
```

Comment:
Messages here match those for posts, but obviously edited to reference messages instead.